src/author.rs

```diff
@@ -51,6 +51,12 @@ impl Author {
     }
 }
 
+/// The set of errors which may occur.
+///
+/// This enum describes the possible errors when parsing the author information.
+/// A valid author information consists of
```

Comment:
I don't like the word information here. Maybe serialized author? Or author string representation?

Reply:
Also the commit message should specify where documentation is added.

Reply:
> I don't like the word information here. Maybe serialized author? Or author string representation?

Okay. :wrench:

Reply:
:wrench: :heavy_check_mark: